src/constraints.js

```diff
@@ -26,59 +26,121 @@ import hasOneNumberGreaterThanInFields from "./constraints/has-one-number-greate
+    singleCodelistValue
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ContainsCodelistValue",
+    containsCodelistValue
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/Codelist",
+    codelist
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ExactValueConstraint",
+    exactValue
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/UriConstraint",
+    validUri
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidDate",
+    validDate
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidDateTime",
+    validDateTime
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidInteger",
+    validInteger
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/PositiveNumber",
+    positiveNumber
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ConceptSchemeConstraint",
+    conceptScheme
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidYear",
+    validYear
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidEmail",
+    validEmail
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidIBAN",
+    validIBAN
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidPhoneNumber",
+    validPhoneNumber
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/VlabelExtraTaxRateOrAmountConstraint",
+    validateExtraTaxRateOrAmount
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidBoolean",
+    validBoolean
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidChildcareSubsidyTable",
+    validChildcareSubsidyTable
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/VlabelSingleInstanceTaxRateOrExtraTaxRate",
+    singleInstanceTaxRateOrExtraTaxRate
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/MaxLength",
+    maxLength
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/ValidEngagementTable",
+    validEngagementTable
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/HasOneNumberGreaterThanInFields",
+    hasOneNumberGreaterThanInFields
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/MatchValues",
+    matchValues
+  );
+  BUILT_IN_VALIDATIONS.set(
+    "http://lblod.data.gift/vocabularies/forms/DateInPast",
+    dateInPast
+  );
+}
+
+registerDefaultValidations();
+
+export function registerCustomValidation(uri, validator) {
+  CUSTOM_VALIDATIONS.set(uri, validator);
+}
+
+export function resetCustomValidations() {
```

Comment:
not sure if actually useful, but it existed for the custom components that i based myself on, so kept it in.